coldfront/core/user/management/commands/dequeue_cluster_account_deactivation_requests.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import sys
+
+from django.core.management import BaseCommand
+
+from coldfront.core.allocation.models import \
+    ClusterAccountDeactivationRequest, \
+    ClusterAccountDeactivationRequestStatusChoice
+from coldfront.core.utils.common import utc_now_offset_aware
+
+
+class Command(BaseCommand):
+    help = 'Command to dequeue ClusterAccountDeactivationRequests after ' \
+           'their waiting period is complete.'
+    logger = logging.getLogger(__name__)
+
+    def add_arguments(self, parser):
+        parser.add_argument('--reason',
```

Comment:
Use [`choices`](https://docs.python.org/3/library/argparse.html#choices)--this will both clean inputs for you and display a list of choices when using `-h`.